bean-validation/3.0/_index.md

```diff
@@ -16,7 +16,7 @@ Jakarta Bean Validation defines a metadata model and API for JavaBean and method
 [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 
 * Maven coordinates
-  * [jakarta.bean-validation:jakarta.bean-validation-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.bean-validation/jakarta.bean-validation-api/3.0.0/jar)
+  * [jakarta.bean-validation:jakarta.bean-validation-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.validation/jakarta.validation-api/3.0.0/jar)
```

Comment:
@starksm64, we're still not consistent here... The link text is using "bean-validation", but the link url is using "validation".  I'm assuming the link text should be updated as well?